src/tests/ftest/deployment/agent_failure.yaml

```diff
@@ -0,0 +1,42 @@
+hosts:
+    test_servers:
+        - server-A
+    test_clients:
+        - client-A
+
+timeout: 300
+
+setup:
+    start_agents_once: False
+
+server_config:
+    name: daos_server
+    servers:
+        bdev_class: nvme
+        bdev_list: ["0000:00:00.0","0000:00:00.1"]
+        scm_class: dcpm
+        scm_list: ["/dev/pmem0"]
+
+pool:
+    size: 100%
+    control_method: dmg
+
+container:
+    type: POSIX
+    control_method: daos
+
+ior:
+    client_processes:
+        np: 1
+    iorflags:
+        flags: "-k -D 300 -v -w -W"
+        api: DFS
+        # This test doesn't work if the IOR runs too fast. In that case, increase
+        # block_size and/or decrease transfer_size. To decrease transfer_size, divide it
+        # by 2, 4, 8, etc.
+        transfer_size: '256K'
+        block_size: '20G'
+        write_x: 4
+        read_x: 2
+        dfs_oclass: "SX"
+        dfs_dir_oclass: "SX"
```

Comment:
We should be using data protection here. Fault management test without data protection won't be much of a meaning.

Reply:
Does that matter with an agent failure? But either way, data protection provides better overall coverage

Reply:
Sure. Makes sense. I will resolve it.